src/analysis/normed_space/operator_norm.lean

```diff
@@ -394,6 +394,16 @@ lemma op_norm_smul_le {𝕜' : Type*} [normed_field 𝕜'] [normed_space 𝕜' F
 instance to_semi_normed_group : semi_normed_group (E →SL[σ₁₂] F) :=
 semi_normed_group.of_core _ ⟨op_norm_zero, λ x y, op_norm_add_le x y, op_norm_neg⟩
 
+lemma nnnorm_def (f : E →SL[σ₁₂] F) : ∥f∥₊ = Inf {c | ∀ x, ∥f x∥₊ ≤ c * ∥x∥₊} :=
+begin
+  ext,
+  rw [nnreal.coe_Inf, coe_nnnorm, norm_def, subtype.coe_image],
+  simp_rw [← nnreal.coe_le_coe, nnreal.coe_mul, coe_nnnorm],
+  -- `mem_set_of_eq` doesn't work here.
+  dunfold set.has_mem has_mem.mem set.mem set_of,
```

Comment:
I've no idea why lean didn't let me apply this lemma, but unfolding wasn't completely awful.